File: example/src/app/Models/GenerateMedicine.ts
export class GenerateMedicine{

    Id :Number
    MedicineName:string
    Dosage:Number
    DosageName:string
    constructor(i:number,m:string,d:Number,dN:string)
    {
        this.Id=i;
        this.MedicineName=m;
        this.Dosage=d;
        this.DosageName=dN;
    }
        
}

File: c#/Models/GenerateMedicine.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models
{
    public class GenerateMedicine
    {
        public int Points { get; set; }
        public long Id { get; set; }
        public string MedicineName { get; set; }
        public int Dosage { get; set; }
        public string DosageName { get; set; }
        public bool Status { get; set; }

        public TimeSpan Time { get; set; }
    }
}
